backend/src/main/java/pw/wp6/avocado_toast/invoker/DatabaseConnection.java

```diff
@@ -22,28 +22,46 @@
             c = DriverManager.getConnection("jdbc:sqlite:file:" + dbPath);
             try (Statement stmt = c.createStatement()) {
                 stmt.execute("PRAGMA foreign_keys = 1");
-                stmt.execute("CREATE TABLE IF NOT EXISTS users\n" +
+                stmt.execute("CREATE TABLE IF NOT EXISTS Transactions\n" +
                         "(\n" +
-                        "  id           INTEGER PRIMARY KEY,\n" +
-                        "  name         TEXT NOT NULL,\n" +
-                        "  username     TEXT UNIQUE NOT NULL,\n" +
-                        "  password     TEXT NOT NULL,\n" +
-                        "  ssn          TEXT NOT NULL,\n" +
-                        "  account_type TEXT NOT NULL\n" +
+                        "  id                   INTEGER     PRIMARY KEY,\n" +
+                        "  merchant             TEXT        NOT NULL,\n" +
+                        "  amount               INTEGER     NOT NULL,\n" +
+                        "  date_time            DATETIME    NOT NULL\n" +
                         ");");
-                stmt.execute("CREATE TABLE IF NOT EXISTS ledger_entries\n" +
+                stmt.execute("CREATE TABLE IF NOT EXISTS Banker\n" +
                         "(\n" +
-                        "  id        INTEGER PRIMARY KEY,\n" +
-                        "  user_id   INTEGER  NOT NULL REFERENCES users (id),\n" +
-                        "  merchant  TEXT     NOT NULL,\n" +
-                        "  amount    DECIMAL  NOT NULL,\n" +
-                        "  date_time DATETIME NOT NULL\n" +
+                        "  id                   INTEGER  PRIMARY KEY,\n" +
+                        "  banker_user_id       INTEGER  NOT NULL REFERENCES Transactions (id),\n" +
+                        "  banker_username      TEXT     UNIQUE NOT NULL,\n" +
+                        "  banker_password      TEXT     NOT NULL,\n" +
+                        "  name                 TEXT     NOT NULL,\n" +
+                        "  account_type         TEXT     NOT NULL\n" +
+                        ");");
+                stmt.execute("CREATE TABLE IF NOT EXISTS Analyst\n" +
+                        "(\n" +
+                        "  id                   INTEGER  PRIMARY KEY,\n" +
+                        "  analyst_user_id      INTEGER  NOT NULL REFERENCES Transactions (id),\n" +
+                        "  analyst_username     TEXT     UNIQUE NOT NULL,\n" +
+                        "  analyst_password     TEXT     NOT NULL,\n" +
+                        "  name                 TEXT     NOT NULL,\n" +
+                        "  account_type         TEXT     NOT NULL\n" +
```

Comment:
The account type column is redundant. There can only be customers in the customer table.